cmd/fleetctl/scripts_test.go

```diff
@@ -45,15 +45,26 @@ func TestRunScriptCommand(t *testing.T) {
 	ds.AppConfigFunc = func(ctx context.Context) (*fleet.AppConfig, error) {
 		return &fleet.AppConfig{ServerSettings: fleet.ServerSettings{ScriptsDisabled: false}}, nil
 	}
+	ds.GetScriptIDByNameFunc = func(ctx context.Context, name string, teamID *uint) (uint, error) {
+		return 1, nil
+	}
+	ds.IsExecutionPendingForHostFunc = func(ctx context.Context, hid uint, scriptID uint) ([]*uint, error) {
+		return []*uint{}, nil
+	}
+	ds.GetScriptContentsFunc = func(ctx context.Context, id uint) ([]byte, error) {
+		return []byte("echo hello world"), nil
+	}
 
 	generateValidPath := func() string {
 		return writeTmpScriptContents(t, "echo hello world", ".sh")
 	}
-	maxChars := strings.Repeat("a", 500001)
+	maxChars := strings.Repeat("a", 10001)
```

Comment:
Just want to double-check that this was intentional, I think @jahzielv recently modified it to 500001?

Reply:
Looks like that was a mistake on my part, I misread the story. looks like if we're passing script contents, we're still limited to 10k chars: 

> For both POST /scripts/run/sync POST /scripts/run return an error if script_contents is specified and the script is over 10k characters. Error message:
Error: Script is too large. 'script_content' is limited to 10,000 characters. To run larger script save it to Fleet and use 'script_name' parameter.